debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/ReplicationConnection.java

```diff
@@ -122,7 +122,7 @@ static String format(long lsn) {
         /**
          * The number of seconds the replication connection should periodically send updates to the server.
          * 
-         * @param statusUpdateIntervalSeconds a number of seconds; must be positive
+         * @param statusUpdateIntervalSeconds a number of seconds; zero or negative disables
```

Comment:
@DuncanSands, so before this change we'd have set up PG's `ChainedLogicalStreamBuilder` with status interval = 0. As per the PG docs this should have disabled the status updates, but apparently it didn't? Or how could this have caused the high CPU usage otherwise?

But there's also an inconsistency with this comment here: when receiving 0 here, we won't pass any status interval any more to PG, causing PG's default of 10 seconds to be applied instead of disabling the status updates. So "zero or negative disables" seems not quite right. But then it seems there is no way to disable the updates given we passed 0 before and apparently that didn't work. Or am I missing something?